### DISCUSSION TITLE
Home Assistant Birdseye card?

### DISCUSSION THREAD

**Discussion by Lockie85** [blakeblackshear/frigate#7415] [Ask A Question]
Hey,

I'm on Frigate version: 0.12.1.

I've seen some posts on this being possible but I'm struggling to understand how. I've got 6 cameras and I'd like to have a birdseye view of them using a frigate card. How can this be achieved? Has anyone seen a guide on this?

Many thanks

**Comment by NickM-27** [comment]
Either using birdseye restream and go2rtc provider in the card or birdseye via jsmpeg provider in the card

**Reply by Lockie85** [reply]
Not sure I'm following sorry. So in my frigate yaml I have:
```
birdseye:
  # Optional: Enable birdseye view (default: shown below)
  enabled: True
  # Optional: Restream birdseye via RTSP (default: shown below)
  # NOTE: Enabling this will set birdseye to run 24/7 which may increase CPU usage somewhat.
  restream: True
```
What do I do now? I don't see Restream in the card, unless I'm missing it.

Many Thanks.

**Reply by NickM-27** [reply]
Reload the home assistant integration, a birdseye camera will be created. Then you add it in the card like any other camera

**Reply by Lockie85** [reply]
I did this but I can't seem to see the camera listed. However, I just found this code, pasted it in and it seems to now display all cameras in birdseye view:
```
type: custom:frigate-card
cameras:
  - frigate:
      camera_name: birdseye
    dependencies:
      all_cameras: true
```

**Reply by Lockie85** [reply]
Restarted Home Assistant and it appeared. Perfect!

Many Thanks for your help.
